test/aes_sky130hs.metrics_limits

```diff
@@ -6,8 +6,13 @@
  ,"worst_slack_max" : "-1.454543597726663"
  ,"tns_max" : "-617.8674409370061"
  ,"clock_skew" : "0.6304365208318368"
+<<<<<<< Updated upstream
  ,"max_slew_violations" : "72"
  ,"max_capacitance_violations" : "6"
+=======
+ ,"max_slew_violations" : "70"
+ ,"max_capacitance_violations" : "5"
+>>>>>>> Stashed changes
```

Comment:
@jjcherry56 seems like an unresolved merge?

Reply:
good catch; I'll push another pr to fix them because I already merged this